ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html

```diff
@@ -89,6 +89,13 @@ <h5>No results</h5>
         "
       ></ess-sort-by-functionality>
     </div>
+   <ng-container  *ngIf="this.type === 'training'">
+    <a href="https://knowledge-hub.eosc-portal.eu">
```

Comment:
Shouldn't we take the url from the config?

Reply:
true, done.